src/config/settings.ts

```diff
@@ -226,6 +250,11 @@ export function init(): Settings {
   try {
     settings = load();
     _isFirstRun = false;
+    // Reset options that shouldn't persist across runs:
```

Comment:
No, because I only needed to check it the first time we load the settings. In fact, `tray.ts` calls `settings.load` later, and if we put the check in `load` and had set it explicitly on the command-line, the second call would turn it off again.